spec/v2/providers/https.spec.ts

```diff
@@ -90,6 +91,7 @@ describe('onRequest', () => {
       {
         ...FULL_OPTIONS,
         region: ['us-west1', 'us-central1'],
+        invoker: ['service-account1', 'service-account2'],
```

Comment:
Don't these need to end with '@'?

Reply:
In the CLI, I've wrote a helper function to take either `service-account`, `service-account@`, or `service-account@project-id.iam.gserviceaccount.com`

Reply:
Updated the unit test to use service accounts that end with a '@'